cmd/emulator/start/start.go

```diff
@@ -65,6 +65,7 @@ type Config struct {
 	ScriptGasLimit            int           `default:"100000" flag:"script-gas-limit" info:"gas limit for scripts"`
 	WithContracts             bool          `default:"false" flag:"contracts" info:"deploy common contracts when emulator starts"`
 	SkipTransactionValidation bool          `default:"false" flag:"skip-tx-validation" info:"skip verification of transaction signatures and sequence numbers"`
+	Host                      string				`default:"0.0.0.0" flag:"host" info:"host to listen on for emulator GRPC/REST/Admin servers"`
```

Comment:
could we default to 127.0.0.1?

Reply:
for sure, only reason I did 0.0.0.0 was so it wouldn't break any existing functionality of the emulator but this is more sensible.